docs/gettingstarted.rst

```diff
@@ -1,4 +1,50 @@
 Getting started
 ===============
 
-FIXME
+To be able to deploy the REANA system, the Kubernetes configuration must be generated. In order to do so, we should do the following.
+
+.. code-block:: sh
+
+   $ python setup.py install
+   $ reana-resources-k8s build-manifests
+
+Once we have done it, we should be able to see the directory `configuration-manifests` with the following structure:
+
+.. code-block:: sh
```

Comment:
Since you are showing more than just a shell script source, it's better to use `code-block:: console` here.  (And everywhere where you show a console session, i.e. shell commands and their output.)